### DISCUSSION TITLE
login-sharif/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#53] [Announcements]
# login-sharif/

اسکریپت لاگین برای شریف، طریقه حل مشکل داکر و البته DNS دستی

https://aprd.ir/login-sharif/

**Comment by Shahedrazavi** [comment]
سلام. خواستم ازتون تشکر کنم بابت این پستتون. خیلی مفید بود و من رو امروز نجات داد! D:

**Comment by mmozaffar** [comment]
سلام. خواستم بابت پستت تشکر کنم. جدیدا اومدم شریف. اصلا لب‌تاپم وصل نمی‌شد. بعد دیدم راست می‌گی مشکل از داکره!! -) -) -)

**Comment by mmozaffar** [comment]
فقط یه نکته رو یادم رفت بگم!
اون آخر اسکریپت رو باس این‌جوری می‌نوشتید:
alias sharif_login1="sharif_login YourUsername Yourpassword"
alias sharif_ip1="sharif_ip YourUsername YourPassword"
alias 1="sharif_login1 && exit"
alias ۱="1"
یعنی برا اون تا دستور هم alias نیازه. از zsh استفاده می‌کنم روی debain . حال برای بقیه والله اعلم

**Reply by pourmand1376** [reply]
ممنون از تذکرت. اصلاحش کردم.

**Comment by MhmRhm** [comment]
Hello. Thanks for the tutorial. I am trying to connect outside of the university from a Linux system. I have installed `sudo apt-get install network-manager-l2tp-gnome` but after a short while my connection drops. Any Ideas?

**Reply by pourmand1376** [reply]
Hey, I don't have access to sharif network since last year. 

I know that there were serious connection issues recently. I don't know if they are resolved or not.
